### PR TITLE
Small user guide update

### PR DESCRIPTION
**Pull request by kelsos**

### CODE REVIEW

File: docs/usage_guide.rst [hunk 1]
```diff
@@ -771,64 +794,122 @@ When both the source identifier and target asset are selected you can press the
...
+   :alt: Price management
+   :align: center
+
+To add a new price you have to press the plus button. This will open the add form.
+
+.. image:: images/price_management_add.png
+   :alt: Adding a new price
+   :align: center
+
+There you can specify the assets the price and the date of the price. Then you can proceed to save the entry.
```
Comment (yabirgb) [inline docs/usage_guide.rst:813]: I believe a comma is needed 
`There you can specify the assets, the price and the date of the price`

File: docs/usage_guide.rst [hunk 2]
```diff
@@ -919,6 +1000,30 @@ Finally you can see a piechart of the distribution of your netvalue across all o
...
+Gitcoin Grants
+***************
+
+As a premium user you can also keep track of your Gitcoin grants and generate reports for them.
+
+.. image:: images/gitcoin_grants.png
+   :alt: Gitcoin grant events
+   :align: center
+
+You can type grant id, select a period you are interested in and press the fetch button to retrieve the events for this
```
Comment (yabirgb) [inline docs/usage_guide.rst:1013]: Here we should mention how to get the grant id. In my case I go to the project page and in the url `https://gitcoin.co/grants/149/rotki` the grant id is before the project name and after grants. I don't know if there is a better way to find it

File: docs/usage_guide.rst [hunk 3]
```diff
@@ -26,10 +26,14 @@ For creating an account press "Create New Account" and provide a username and a
   - **Username**: it is just an identifier for your database; a local user.
   - **Password**: :red:`Do not forget this password`. It is used to encrypt all your local files.
 
-If you have purchased a premium subscription you can also add the **API Key** and the **secret** here. See the section :ref:`sync-data-with-rotki-server` to know how to sync your data with Roki Server (this option is disabled by default).
+If you want to restore an account using premium sync during the account creation, then you can **Enable premium**,
+and insert your **API Key** and the **secret** here.
+
+For a completely new account it is suggest to add your premium API key and secret using :ref:`set-up-rotki-premium`
```
Comment (LefterisJP) [inline docs/usage_guide.rst:32]:
```suggestion
For a completely new account it is suggested to add your premium API key and secret using :ref:`set-up-rotki-premium`
```

File: docs/usage_guide.rst [hunk 4]
```diff
@@ -26,10 +26,14 @@ For creating an account press "Create New Account" and provide a username and a
...
+after login in with the new account.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:33]:
```suggestion
after logging in with the new account.
```

File: docs/usage_guide.rst [hunk 5]
```diff
@@ -562,11 +581,15 @@ rotki allows to track balances of blockchain accounts.
 
 To add or modify an account navigate to the "Blockchain Balances" sub-page and click the large "+" icon. Now choose the blockchain on which you want to add an account (for now only Bitcoin, Ethereum and Kusama chains are supported). Then type or paste the address in the "Account" textbox and press the "Save" Button.
 
-.. image:: images/add_blockchain_balance.gif
+.. image:: images/add_blockchain_account.png
    :alt: Add a blockchain account
    :align: center
 
-To stop tracking a particular account scroll down to the accounts tables and click the "Delete" icon (trashcan) under Actions.
+To stop tracking one or more accounts you can check corresponding box in the accounts table and click the "Delete" button.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:588]:
```suggestion
To stop tracking one or more accounts you can check the corresponding box in the accounts table and click the "Delete" button.
```

File: docs/usage_guide.rst (hunk 1 above)
Comment (LefterisJP) [reply docs/usage_guide.rst:813]:
```suggestion
There you can specify the assets, the price and the date of the price. Then you can proceed to save the entry.
```

File: docs/usage_guide.rst [hunk 6]
```diff
@@ -771,64 +794,122 @@ When both the source identifier and target asset are selected you can press the
 On a successful merge you will be notified to to either re-login or refresh the balances manually to see the changes
 on the frontend.
 
+Adding missing prices
+**********************
+
+Some times rotki might be unable to retrieve a historical price for an asset from CoinGecko or CryptoCompare.
+In this case you can use the price management interface to insert your own entries of prices.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:801]:
```suggestion
In this case you can use the price management interface to insert your own price entries.
```

File: docs/usage_guide.rst [hunk 7]
```diff
@@ -771,64 +794,122 @@ When both the source identifier and target asset are selected you can press the
...
+Deposits
 ===========
 
-In the lending section you can see the status of your DAI in the DAI Savings Rate (DSR). The accounts that use DSR are auto-detected from your given blockchain accounts. You can see how much DAI you have locked over all of your accounts and how much DAI is locked for each account in the DSR.
+In the deposits section you can see the status of your different Defi lending protocols.
+You can see the status of your Yearn Vaults, Aave lending, Compound supply along with
+your DAI in MakerDAO DSR.
 
-.. image:: images/sc_dsr_nonpremium_all.png
+The accounts that can be auto-detected from your given blockchain accounts.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:837]:
```suggestion
The accounts are auto-detected from your given blockchain accounts.
```

File: docs/usage_guide.rst [hunk 8]
```diff
@@ -771,64 +794,122 @@ When both the source identifier and target asset are selected you can press the
...
+However you can manage the different modules and addresses that are queried to make the retrieval faster.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:838]:
```suggestion
However you can, and most probably should, manage the different modules and addresses that are queried to make the retrieval faster.
```

File: docs/usage_guide.rst [hunk 9]
```diff
@@ -771,64 +794,122 @@ When both the source identifier and target asset are selected you can press the
...
+for each account across the different protocols.
+
+.. image:: images/sc_decentralized_deposits_no_premium.png
    :alt: DSR without premium
    :align: center
 
-You can also filter by account and see how much DAI is locked in the DSR for each account.
-
-If you have a premium subscription you can also see how much DAI you have gained in total or over each account, the history of deposits/withdrawals and how much DAI was earned at the point in time for each action.
+You can also filter by account and protocol and you can see how the assets locked in the various protocols.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:848]:
```suggestion
You can also filter by account and protocol and you can see how the assets are locked in the various protocols.
```

File: docs/usage_guide.rst [hunk 10]
```diff
@@ -771,64 +794,122 @@ When both the source identifier and target asset are selected you can press the
...
 
-Below you can see a small demonstration of the usage of DSR by a premium account.
+.. image:: images/sc_decentralized_deposits_yearn.png
+  :alt: Defi Deposits Yearn Vaults
+  :align: center
 
-.. image:: images/dsr_premium_demo.gif
-   :alt: DSR premium demo
-   :align: center
+You can also a details list of historical actions performed in the supported Defi protocols such as deposits withdrawals etc.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:860]:
```suggestion
You can also get a detailed list of historical actions performed in the supported Defi protocols such as deposits withdrawals etc.
```

File: docs/usage_guide.rst [hunk 11]
```diff
@@ -771,64 +794,122 @@ When both the source identifier and target asset are selected you can press the
...
+
+.. image:: images/sc_decentralized_deposits_history.png
+  :alt: Defi Deposits history
+  :align: center
+
+Finally you need to have premium in order for the total amount of earned assets in a given time period to be counted in the profit/loss report.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:866]:
```suggestion
Finally you need to have a premium subscription in order for the total amount of earned or lost value per asset in a given time period to be counted in the profit/loss report.
```

File: docs/usage_guide.rst (hunk 2 above)
Comment (LefterisJP) [reply docs/usage_guide.rst:1013]:
```suggestion
You need to first find the id ofthe grant you are interested in. When you visit the grant, the id is what comes after the ``grants/`` part of the url. For example for rotki (https://gitcoin.co/grants/149/rotki) the id is ``149``.

 You type the grant id, select a period you are interested in and press the fetch button to retrieve the events for this
```

File: docs/usage_guide.rst [hunk 12]
```diff
@@ -919,6 +1000,30 @@ Finally you can see a piechart of the distribution of your netvalue across all o
...
+
+.. image:: images/gitcoin_grants_report.png
+   :alt: Gitcoin grant events
+   :align: center
+
+With the events retrieved you can also generate reports for specific periods. In these reports you can see a breakdown
+of the assets received for the specified grant over the selected period.
+
+.. warning::
+    The report depends an the already fetched events. If any events are missing during the selected report period
```
Comment (LefterisJP) [inline docs/usage_guide.rst:1024]:
```suggestion
    The report depends on the already fetched events. If any events are missing during the selected report period
```

**Review by LefterisJP — APPROVED** [review]
Generally looks good. Thank you very much. I left some wording changes, addition and typos, which I will apply directly.

**Pull request merged**